src/ledger/LedgerState.cpp

```diff
@@ -1364,6 +1364,11 @@ std::shared_ptr<LedgerEntry const>
 LedgerStateRoot::Impl::getBestOffer(Asset const& buying, Asset const& selling,
                                     std::set<LedgerKey>& exclude)
 {
+    // Note: Elements of mBestOffersCache are properly sorted lists of the best
```

Comment:
I know we don't have anything like that right now, but we could have post conditions enabled in debug builds: `dbgAssert( isSorted(...) )`